Cargo.lock

```diff
@@ -1152,6 +1152,27 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "bindgen"
```

Comment:
Do we need large dependencies like `rocksdb` (and its dependencies `bindgen` and `clang-sys`)?
Is there a set of features that will minimise the extra dependencies?

Reply:
Thanks for the catch!